georocket-server/src/test/java/io/georocket/index/elasticsearch/RemoteElasticsearchClientTest.java

```diff
@@ -65,6 +66,20 @@
               .put("error", new JsonObject()
                 .put("type", "mapper_parsing_exception")
                 .put("reason", "Field name [nam.e] cannot contain '.'")))));
+
+  private static final JsonObject SETTINGS = new JsonObject("{\n" +
+      "  \"analysis\" : {\n" +
+      "    \"analyzer\":{\n" +
+      "      \"content\":{\n" +
+      "        \"type\":\"custom\",\n" +
+      "        \"tokenizer\":\"whitespace\"\n" +
+      "      }\n" +
+      "    }\n" +
+      "  }\n" +
+      "}");
+
+  private static final JsonObject ACKNOWLEDGED = new JsonObject()
```

Comment:
Please use this field in all cases where an acknowledgment is returned in this class.